WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -31,10 +44,19 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        updateSortingOption(.popularity, reloadCards: false)
+
         ReaderWelcomeBanner.displayIfNeeded(in: tableView)
         tableView.register(ReaderTopicsCardCell.self, forCellReuseIdentifier: readerCardTopicsIdentifier)
         tableView.register(ReaderSitesCardCell.self, forCellReuseIdentifier: readerCardSitesIdentifier)
 
+        tableView.tableHeaderView = sortingButton
+
+        NSLayoutConstraint.activate([
+            sortingButton.widthAnchor.constraint(equalTo: tableView.widthAnchor),
+        ])
```

Comment:
can we group the sorting option tasks in a method called here, to leave `viewDidLoad` cleaner?